src/js/dp/copy-link.js

```diff
@@ -34,7 +34,7 @@ document.addEventListener('DOMContentLoaded', () => {
     navigator.clipboard
       .writeText(window.location.href)
       .then(success(btnText), failure(btnText))
-      .catch((error) => console.error(error));
+      .catch((error) => console.error(error)); // eslint-disable-line
```

Comment:
possible debate about this to be had.

Reply:
agreed a global config change. will revert this and push a new branch with the change